spyql/processor.py

```diff
@@ -167,13 +191,23 @@ def prepare_expression(self, expr):
         Replaces identifiers (column names) in sql expressions by references to
         `_values` and put (quoted) strings back
         """
-        if expr == "*":
-            return self.col_values_exprs
-
         if isinstance(expr, int):
             # special case: expression is out col number (1-based)
             return [f"_res[{expr-1}]"]  # reuses existing result
 
+        if expr == "*":
+            return self.col_values_exprs
+
+        except_pattern = re.compile(r"\*\s*EXCEPT\s*\((.*)\)", re.IGNORECASE)
+        except_match = re.search(except_pattern, expr)
+        if except_match:
+            excpt = [x.strip() for x in except_match.group(1).split(",")]
```

Comment:
Again, moving this to the parsing stage would make this simpler. Otherwise, it is nicely done :-)